src/rate-limit-view/rate-limit-view.ts

```diff
@@ -0,0 +1,88 @@
+import COLORS from "../data/rate-limit-view-colors.json";
+
+interface RateLimitColors {
+	negative: string,
+	positive: string[]
+}
+
+const RATE_LIMIT_COLORS: RateLimitColors = COLORS;
+const RATE_LIMIT = 333;
+const COLOR_INTERVALL = 20;
+const COLOR_MAX_MS = RATE_LIMIT_COLORS.positive.length * COLOR_INTERVALL;
+
+export class RateLimitView {
+	constructor(Elements: any, Options: any) {
+		Elements.$body.attr("data-RateLimitView", "Disabled");
```

Comment:
Element constants should be stored in `elements.ts` instead of using `Elements`.

Reply:
From what I can tell the element is used to store data that is never accessed. I'll instead remove this completely.